eng/scripts/tests/Invoke-GenerateAndBuildV2.tests.ps1

```diff
@@ -0,0 +1,52 @@
+#Requires -Version 7.0
```

Comment:
It would be helpful to add a comment here or in a readme with a quick note about how to run these, e.g. https://github.com/Azure/azure-sdk-tools/blob/b6f98d351ceac565eeb2cbe23f762e123f120ef6/tools/asset-sync/contributing.md#using-pester

Reply:
Add comment ahead of the test scripts.